src/zenml/orchestrators/utils.py

```diff
@@ -305,3 +308,95 @@ def _get_model_versions_from_artifacts(
             else:
                 break
     return models
+
+
+class artifact_store_handler:
```

Comment:
I think the name does not signal at all what this does. In what way does it handle my artifact store for me?

Maybe `register_artifact_store_filesystem`?

Reply:
Solid point. Renamed the context manager.